src/main.js

```diff
@@ -94,12 +94,14 @@ class Entry {
    *@param {string|null} envType type of variable e.g. url
    *@param {Array|null} envChoices given restricted choices e.g. [data,info,1]
    *@param {string|null} envDefaultVal given default value e.g. data
+   *@param {string} envComment given comment using the # symbol
    */
-  constructor(envName, envType, envChoices, envDefaultVal) {
+  constructor(envName, envType, envChoices, envDefaultVal, envComment) {
```

Comment:
I think we can rename all arguments to simpler names 😄. What about just `name`, `type`, `choices`, `defaultValue` and `comment`?

The `env...` prefix does not provide anything useful to the reader.

**P.S.:** Do not forget to update the JSDoc as well